src/components/ec-filter-popover/ec-filter-popover.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <div
+    class="ec-filter-popover"
+    data-test="ec-filter-popover"
+  >
+    <ec-popover
+      popover-class="ec-popover"
+      placement="bottom"
+      trigger="click"
+      level="modal"
```

Comment:
Why it must be a modal?

Reply:
in snapshot was `level=""` though that I needed to specify something and thought modal was the pertinent one

Reply:
We should override the level only if it's really needed. some components that are using ec-popover might be used inside modal dialog, e.g. ec-dropdown. If ec-dropdown is not used inside of a modal, modal's z-index should be greater than z-index of ec-popover in ec-dropdown. if ec-dropdown is used inside of a modal, then z-index of modal should be smaller than z-index of ec-popover. That's why we have those levels.

In this case, ec-filter-popover will never be used inside of a modal, so it should always have default level (not set at all) so the modal which has higher z-index can be displayed over the filter.

Just don't set this value.